src/gossip/service.zig

```diff
@@ -150,7 +133,7 @@ pub const GossipService = struct {
     ) error{ OutOfMemory, GossipAddrUnspecified, SocketCreateFailed, SocketBindFailed, SocketSetTimeoutFailed }!Self {
         var packet_incoming_channel = PacketBatchChannel.init(allocator, 10000);
         var packet_outgoing_channel = PacketBatchChannel.init(allocator, 10000);
-        var verified_incoming_channel = ProtocolChannel.init(allocator, 10000);
+        var verified_incoming_channel = GossipMessageChannel.init(allocator, 10000);
```

Comment:
I'm not suggesting a change here, since it's not in scope, but I just wanted to mention a minor nitpick... I feel that these channel type aliases make it more difficult to navigate the code. Navigating to the definition of the data type being passed in the channel is slightly more complicated because vscode just navigates directly to `Channel` from here. I also feel that `Channel(GossipMessage)` is clearer, in terms of readability.

Reply:
yeah agreed - hopefully zls makes a fix for this soon - even though its out of scope, i still made the fix to help clean up the code - i think theres a threshold of how often a type is used before its defined as a seperate typedef

(86c7d3a)

Reply:
> i think theres a threshold of how often a type is used before its defined as a seperate typedef

I agree, and I think it needs to satisfy some other requirements too, to justify the layer of indirection. Like does it improve or hurt readability. Is it somewhere we can benefit from abstraction, or is it better to be transparent. etc.